mujoco_warp/_src/smooth.py

```diff
@@ -159,15 +159,22 @@ def _geom_local_to_global(
   xpos_in: wp.array2d(dtype=wp.vec3),
   xquat_in: wp.array2d(dtype=wp.quat),
   # Data out:
+  geom_sameframe_out: wp.array(dtype=int),
```

Comment:
We discussed with Miles that in the future we may want to turn on differentiability, and to do so we may need to explicitly track in/out variables.  `geom_sameframe` is an in param and an out param, so we should repeat it, once under `# Data in:` and once under `# Data out:`